src/site/content/en/mini-apps/what-are-super-apps/index.md

```diff
@@ -0,0 +1,88 @@
+---
+layout: post
+title: What are super apps?
+authors:
+  - thomassteiner
+date:  2021-02-05
+# updated: 2021-01-25
+description: |
+  This chapter introduces the concept of super apps and presents the major players.
+tags:
+  - mini-apps
+---
+
+## Welcome to mini apps
+
+When you look at the applications on your phone, you probably have specific apps for specific tasks.
+You might have a banking app. You might have an app for buying public transit tickets. Likely you
+have an app for getting directions, etc. In this post, I introduce you to the concept of a different
+kind of app, namely mini apps, sometimes also referred to as mini programs or applets. I first
+provide some background on the various mini app platforms and their developer experience, and then
+focus on things the Web can learn from mini apps. But before I start writing about mini apps, I
+first need to talk about super apps.
```

Comment:
Would be good to have a screenshot here. Since this is the entrypoint of the entire series.

Reply:
Added.